lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
```

Comment:
This wasn't covered in class but i usally do anything involving process.argv in the main part of my program. ill parse it and store it in a variable then pass that variable into any of my modules that can use the data... 

it makes the program easier to read :)